staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -2124,8 +2131,8 @@ func TestListInconsistentContinuation(t *testing.T) {
 		t.Fatalf("Unexpected continuation token set")
 	}
 	expectNoDiff(t, "incorrect third page", []example.Pod{*preset[2].storedObj}, out.Items)
-	if out.ResourceVersion != lastRVString {
```

Comment:
`continueFromThirdItem` encodes a specific rv, so this should exactly match the `ResourceVersion` in the `inconsistentContinueFromSecondItem` list, right? Can we check against that and keep this check exact?